dev/custom-pages/homepage.scss

```diff
@@ -270,8 +269,29 @@ $color-linkedin: #0077b5;
     }
 
     .padded-bottom {
-      padding-bottom: 70px;
+      padding-bottom: 47px;
     }
+
+    @media (max-width: $screen-md-max) {
```

Comment:
Avoid nested selectors within media queries when using sass--something like
```
img {
  @media (max-width: $screen-md-max) {
    top: 73px;
  }
}
```
is preferred (per-selector media queries).